_data/patcher-landing-page-tabs-terraform-upgrade.yml

```diff
@@ -0,0 +1,7 @@
+- id: pr
+  title: PR
+  content: https://i.imgur.com/XDAl1Ew.png
```

Comment:
This first screenshot is too small, need to figure out how to make it better.

Reply:
One option is to record a `.gif` of you scrolling and panning around a pull request with the title "Upgrade to Terraform 1.0." When recording the `.gif`, you set the window size to roughly the space available here (looks like 410px x 310px), so then there's no resizing at all.

Reply:
Done in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/583/commits/8a1c45c8925b25282507b7eb05527505cf620436.